docs/src/tutorial/genomic-surveillance.rst

```diff
@@ -0,0 +1,158 @@
+Run the workflow using a genomic surveillance configuration
+===========================================================
+
+In this tutorial, you will replace the custom contextual data with meaningful focal data and use a configuration commonly found in genomic surveillance applications.
+
+.. contents:: Table of Contents
+   :local:
+
+Prerequisites
+-------------
+
+1. :doc:`custom-data`. These instructions will introduce the workflow configuration that is modified in this tutorial.
+
+Setup
+-----
+
+If you are not already there, change directory to the ``ncov`` directory:
+
+   .. code:: text
+
+      cd ncov
+
+Curate data from GISAID
+-----------------------
+
+.. warning::
+
+   These steps require a GISAID account. `Register <https://www.gisaid.org/registration/register/>`__ if you do not have an account yet. However, registration may take a few days. Follow `alternative data preparation methods <../guides/data-prep.html>`__ if you wish to continue this tutorial in the meantime.
+
+We will download a custom focal set from GISAID's EpiCoV database.
+
+1. Navigate to `GISAID <https://www.gisaid.org/>`__, **Login**, and go to **EpiCoV** > **Search**.
+
+   .. image:: ../images/gisaid-epicov-search.png
+      :width: 400
+      :alt: GISAID EpiCoV Search
+
+2. Select a custom set of records. As an example, this guide will look for all records that pass the following criteria:
+
+   1. From Idaho
+   2. Collected between ``2022-02-15`` and ``2022-03-15``
```

Comment:
I'm not sure how to best handle this, but these dates will quickly get out of date with the reference data we use in the rest of the tutorial and we're not guaranteed to always get ~200 records with this search when future data submitters add more records for these dates.

What if we asked users to just set the minimum collection date to the last month or two, filtering until they get ~200 records?

Reply:
The selection here is mostly for example purposes, I think we should encourage the user to pick something relevant to their own interest given:

1. Best results with ~200 sequences
2. Max 5,000 sequences
3. More sequences = longer run time

I tried to make this point in the `.. note::` below the screenshot, but can change the step to say something like:

> Select a custom set of records. We recommend a set of ~200 sequences as a good balance between a meaningful amount of data and run time. Note that more records will take the workflow longer to run, and the GISAID web interface has a maximum of 5,000 records per download.
> 
> As an example, this guide will look for all records that pass the following criteria:
> ...

Reply:
Probably another point worth discussing with the broader group. There could be two different sets of users:

1. Those who want to run the tutorial from start to end by just running commands. These users may just want to get a sense of how these types of analyses can be configured through this worked example before they embark on their own analyses.
2. Those who want to use the tutorial as a guide to setup their own genomic epi analysis.

Thinking back to [the documentation system](https://documentation.divio.com/tutorials/) that we've previously used to guide our docs, a tutorial like this should focus on users in the first group. They are learning by doing first. Then, they can use what they've learned to implement their own genomic epi analysis.

I think this suggests sticking with a specific example like Idaho, instead of asking users to curate a new dataset. The only issue could be that the fixed time window of Idaho data will not change while the reference data we use will continue to move forward in time. Maybe this is a non-issue since the reference data will always including genomes from the history of the pandemic.

That said, I'm also not sure we _need_ the reference data to keep updating through time. I thought the original goal of the reference data was to provide representative genomes for this tutorial. The fact that we generate these daily with the other ncov builds is an implementation detail that has nothing to do with this original goal.

Reply:
The "Customize this example" section mentioned above has been moved to a new ["Next steps" page](https://nextstrain--894.org.readthedocs.build/projects/ncov/en/894/tutorial/next-steps.html). 334a0d087e3bc1ffde0784539610e7ba474a2af0